test/e2e/config/azure-dev.yaml

```diff
@@ -67,6 +67,9 @@ variables:
   EXP_AKS: "true"
   EXP_MACHINE_POOL: "true"
   EXP_CLUSTER_RESOURCE_SET: "true"
+  CONFORMANCE_CI_ARTIFACTS_KUBERNETES_VERSION: "v1.18.8"
```

Comment:
```suggestion
  CONFORMANCE_CI_ARTIFACTS_KUBERNETES_VERSION: "v1.18.10"
```

Reply:
nevermind, I'll update this in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1005